packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -66,7 +66,7 @@ function NavItem({items, position, ...props}) {
       <ul className="dropdown__menu">
         {items.map((linkItemInner, i) => (
           <li key={i}>
-            <NavLink className="navbar__item navbar__link" {...linkItemInner} />
```

Comment:
Need to change this otherwise the carets will also show up for the dropdown items.

Reply:
Uh-huh, so for dropdown items we need to use _only one_ CSS class of `dropdown__link`. Then we need to add an active dropdown item class to Infima and then specify it in the NavBar component. Right? I will fix it.

Reply:
The fix is ready, I'm waiting for the new version of Infima, which will define CSS class for active dropdown item.